src/task2/palindrome.py

```diff
@@ -0,0 +1,19 @@
+"""Find palindrome
+
+Check whether a number is palindrome or Not.
+Input string contains only numbers.
+"""
+
+
+number_to_check = input("Enter a number: ")
+reversed_number = ""
+number_to_check_len = len(number_to_check)
+
+while number_to_check_len:
```

Comment:
work with arithmetic operators and numbers should be used according to task description